app/models/user.rb

```diff
@@ -59,7 +59,7 @@ def by_conference(conference)
 
   has_many :openids
 
-  attr_accessor :login
+  attr_accessor :login, :invitation_message
```

Comment:
@Ana06 because we need custom messages for users(example: to become booth responsible or to become organizer etc). So I thought having an attribute will be beneficial for any custom message invitation.

Reply:
why don't you pass it as a parameter in `invite`? The invitation doesn't belong to the user model.

Reply:
I will try it :)